android/KMAPro/kMAPro/src/main/assets/svg/banner.html

```diff
@@ -0,0 +1,10 @@
+<!-- Custom banner theme -->
+<div style="background: white; width: 100%; height: 100%; position: absolute; left: 0; top: 0">
+  <!--<div style="height: 1px; left: 0; position: absolute; top: 0; width: 100%; background: black"></div>-->
+  <img src="$BANNER" style="height:100%; background: white; display: block">
```

Comment:
I'm no CSS wizard, but...

```suggestion
<div style="background: white; width: 100%; height: 100%; position: absolute; left: 0; top: 0; box-sizing: border-box">
  <!--<div style="height: 1px; left: 0; position: absolute; top: 0; width: 100%; background: black"></div>-->
  <div style="width: 100%; height: 93%; position: absolute; left: 0; top: 0; box-sizing: border-box; padding: 5px>
    <img src="$BANNER" style="height:100%; background: white; display: block">
  </div>
```

Something like this should help us shrink the image.

`box-sizing: border-box`:  ensures that addition of borders and padding counts as part of the width & height's "100%" spec - they thus subtract from the space allotted to nested contents - that SVG.

Height: 93%:  so that the SVG stays entirely above the tri-color line.  100% might actually be fine, though.

Just spitballing an idea, but you may wish to replace `padding: 5px` with `padding: $PADDING` and do a replacement that's based upon the device resolution or similar, if it turns out we need different padding amounts for different devices.

Reply:
Also, a small warning before you consider getting fancy: https://css-tricks.com/oh-hey-padding-percentage-is-based-on-the-parent-elements-width/

Padding percentage doesn't work ideally for us here.

Reply:
I'll update the user-testing screenshots with some updated styling (`display: flex;` and `margin: auto 2%`)